lib/resources/example_json/r4_examples_relatedperson.rb

```diff
@@ -221,13 +429,30 @@ module Resources
       },
       'relationship': [
         {
-          'id': 'CI-97982588-0-0',
+          'id': 'CI-97982588-0',
           'extension': [
             {
               'valuePeriod': {
                 'start': '2016-12-08T21:53:53.000Z'
               },
               'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/period'
+            },
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40',
+                    'code': '156'
+                  },
+                  {
+                    'system': 'http://terminology.hl7.org/CodeSystem/v3-RoleCode',
+                    'code': 'MTH',
+                    'display': 'Mother',
+                    'userSelected': false
+                  }
```

Comment:
Do we not return a text in this coding?

Reply:
added here and other places applicable https://github.com/cerner/fhir.cerner.com/pull/644/commits/8826890486713f37b02cddcff2f7f3870f42cb16#diff-133ac83b73a8a88b430a777cf1700b9010258bd96bc4ff69a51350091f73643eR468

Reply:
+1